src/platform/macos/events_loop.rs

```diff
@@ -544,6 +544,67 @@ impl Proxy {
     }
 }
 
+pub fn windows_key_code_for_char_code(c: char) -> Option<c_ushort> {
+    // We only translate keys that are affected by keyboard layout.
+    //
+    // Note that since keys are translated in a somewhat "dumb" way (reading character)
+    // there is a concern that some combination, i.e. Cmd+char, causes the wrong
+    // letter to be received, and so we receive the wrong key.
```

Comment:
Have you found any cases where this actually happens?

Reply:
No, I haven't been able to personally observe this.

My concern is in things like these: https://www.webnots.com/option-or-alt-key-shortcuts-to-insert-symbols-in-mac-os-x/.  If one of those codes resulted in a match for `char_to_keycode()` then the key could potentially be incorrect.  This doesn't seem to happen right now, but I don't think it's far fetched to be concerned about it -- what if these codes become user-configurable?
I'm also not sure how those codes work in languages other than english, but maybe I'm going out of scope there.

Reply:
Ah, that makes sense.

> but maybe I'm going out of scope there.

Supporting other languages is very much in scope. From a cursory look, the Hiragana input method uses the same altcodes as the U.S. input method. On 390 Sebulshik, there don't seem to be any altcodes at all... it's probably worth checking out [`InputMethodKit`](https://developer.apple.com/documentation/inputmethodkit?language=objc) to see if there's an API provided for specifying altcodes, since if not, I'm inclined to assume there's only one set of possible values.

Either way, I'm not too worried about this. It doesn't seem likely that Apple will make them user-configurable any time soon, though that's just my gut feeling on this.